engine/updategraph/src/main/java/io/deephaven/engine/updategraph/UpdateGraphProcessor.java

```diff
@@ -1810,7 +1807,7 @@ private UpdateGraphProcessorThreadFactory(@NotNull final ThreadGroup threadGroup
         public Thread newThread(@NotNull final Runnable r) {
             return super.newThread(() -> {
                 configureRefreshThread();
-                r.run();
+                ThreadInitializationFactory.wrapRunnable(r).run();
```

Comment:
This pattern is inconsistent with the other usages I've seen, and I think I prefer the other one. That is, I think this should look like:
```
return super.newThread(ThreadInitializationFactory.wrapRunnable(() -> {
                configureRefreshThread();
                r.run();
```